core/ui/src/androidMain/kotlin/io/github/droidkaigi/confsched2023/ui/compositionlocal/LocalClockProvider.kt

```diff
@@ -0,0 +1,9 @@
+package io.github.droidkaigi.confsched2023.ui.compositionlocal
+
+import androidx.compose.runtime.compositionLocalOf
+import kotlinx.datetime.Clock
+
+@Suppress("CompositionLocalAllowList")
+val LocalClockProvider = compositionLocalOf<Clock> {
```

Comment:
IMO, using `staticCompositionLocalOf` would be better performance.

Reply:
Thank you for the advice! Let me check the function and replace with it!